content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -111,6 +111,8 @@ Kubernetes Volumes enable complex applications with data persistence and Pod vol
 * [azureFile](/docs/concepts/storage/volumes/#azurefile)
 * [gcePersistentDisk](/docs/concepts/storage/volumes/#gcepersistentdisk)
 
+Starting with v1.16, Alpha support for CSI plugins will be introduced in Windows nodes.
```

Comment:
(In case anyone doesn't spot that this is a placeholder commit), the final text should
[avoid making statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future).